pkg/topology/topology.go

```diff
@@ -213,15 +213,47 @@ func FindSysFsDevice(dev string) (string, error) {
 		}
 	}
 
-	major := unix.Major(rdev)
-	minor := unix.Minor(rdev)
+	major := int64(unix.Major(rdev))
+	minor := int64(unix.Minor(rdev))
```

Comment:
isn't it uint8 or uint16 in datatypes?

Reply:
No. IIRC `uint32` in `x/sys/unix` and `int64` in the OCI Spec.